bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/ip.xml

```diff
@@ -73,29 +73,60 @@
 				<description>Seconds between connection retries when KNX link has been lost, 0 means never retry, minimum 30s</description>
 				<default>60</default>
 			</parameter>
+			<parameter name="keyringFile" type="text" groupName="knxsecure">
+				<label>Keyring file</label>
+				<description>Keyring file exported from ETS and placed in openHAB config/misc folder, e.g.
+					knx.knxkeys. This file is
+					mandatory to decode secure GAs. It can provide settings
```

Comment:
GAs ?

Reply:
GAs = Group Addresses
It’s a KNX specific term and I’m pretty sure the users of the KNX binding should understand it, but probably would be good to write it out.